knowledge-base/set-culture-of-date-picker-in-angular-viewer.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Change the Culture of Kendo DatePicker Widget in the Parameters Area of the Angular Report Viewer
+description: How to Set Culture for Kendo Widgets used in Angular Report Viewer
+type: how-to
+page_title: Localize the Angular Viewer Kendo Widgets
+slug: set-culture-of-date-picker-in-angular-viewer
+position: 
+tags: 
+ticketid: 1459450
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The article describes how to change the culture of the Kendo widgets used to build the Angular Report Viewer.
+
+## Solution
+The general solution of the problem for localizing the DatePicker of the 
+[pure Html5 Report Viewer](https://docs.telerik.com/reporting/html5-report-viewer) is described in the 
+[Change the culture of Kendo DatePicker widget in the parameters area of the HTML5 Report Viewer](./how-to-change-the-culture-of-the-datepicker-in-html5-report-viewer) 
+article. The Angular viewer is a wrapper of the pure Html5 viewer that utilizes Kendo UI for jQuery components. For that reason, 
```

Comment:
I think it better to use widgets here -> utilizes Kendo UI for jQuery widgets. This is the way kendo refer to them in their [documentation ](https://docs.telerik.com/kendo-ui/introduction). 'Components' is something specific for the frameworks like Angular, View, Reac :)